dbtsl/api/graphql/client/base.py

```diff
@@ -28,14 +29,22 @@ class BaseGraphQLClient(Generic[TTransport, TSession]):
 
     PROTOCOL = GraphQLProtocol
     DEFAULT_URL_FORMAT = env.DEFAULT_GRAPHQL_URL_FORMAT
+    DEFAULT_TIMEOUT = TimeoutOptions(
+        total_timeout=30,
```

Comment:
30 seconds seems a little low... I wonder if we should look into some datadog logs to figure out the p75 or something?